run-platform.sh

```diff
@@ -169,13 +169,13 @@ setup_env() {
         if [[ "$OSTYPE" == "darwin"* ]]; then
           sed -i '' "s/DEFAULT_AUTH_USERNAME.*/DEFAULT_AUTH_USERNAME=\"$DEFAULT_AUTH_KEY\"/" $env_path
           sed -i '' "s/DEFAULT_AUTH_PASSWORD.*/DEFAULT_AUTH_PASSWORD=\"$DEFAULT_AUTH_KEY\"/" $env_path
-          sed -i '' "s/SYSTEM_ADMIN_USERNAME.*/SYSTEM_ADMIN_USERNAME=\"$DEFAULT_AUTH_KEY\"/" $env_path
-          sed -i '' "s/SYSTEM_ADMIN_PASSWORD.*/SYSTEM_ADMIN_PASSWORD=\"$DEFAULT_AUTH_KEY\"/" $env_path
+#          sed -i '' "s/SYSTEM_ADMIN_USERNAME.*/SYSTEM_ADMIN_USERNAME=\"$DEFAULT_AUTH_KEY\"/" $env_path
```

Comment:
@VikashPratheepan rather than commenting this out, can you set the `SYSTEM_ADMIN_USERNAME` and `SYSTEM_ADMIN_PASSWORD` to `$DEFAULT_ADMIN_AUTH_KEY`. And the `DEFAULT_ADMIN_AUTH_KEY` can be set to `admin` itself - since we've done that in the sample.env file

Reply:
I didn't suggest this because we are doing a quick fix. Let a developer take up the change to include this env if needed afterwards